### PR TITLE
[CI] Add schedule event trigger to binary build workflow

### PR DESCRIPTION
**Pull request by langong347**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #345
* #344
* __->__ #343

Summary:

Add schedule event trigger to automate the nightly binary build.

Test Plan:

Can't really test this as it's a scheduled event.
Ran a manual trigger on this PR as a proxy for correctness though -- find the latest successful run [here](https://github.com/facebookresearch/multimodal/actions/workflows/test_build.yml).

Differential Revision: [D39993621](https://our.internmc.facebook.com/intern/diff/D39993621)

### CODE REVIEW

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39993621).

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39993621).

File: .github/workflows/test_build.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: Build/Push Nightly Binary
 
 on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
```
Comment (ankitade) [inline .github/workflows/test_build.yml:6]: add what this means to the comment => every day at x pm etc

**Review by ankitade — APPROVED** [review]

File: .github/workflows/test_build.yml (hunk 1 above)
Comment (langong347) [reply .github/workflows/test_build.yml:6]: Those can be looked up on Github docs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

**Comment by langong347** [comment]
@langong347 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39993621).

**Pull request closed without merging**